cmd/minikube/cmd/delete.go

```diff
@@ -90,26 +90,31 @@ func init() {
 }
 
 // shotgun cleanup to delete orphaned docker container data
-func deleteContainersAndVolumes() {
-	if _, err := exec.LookPath(oci.Docker); err != nil {
-		glog.Infof("skipping deleteContainersAndVolumes for %s: %v", oci.Docker, err)
+func deleteContainersAndVolumes(ociBin string) {
+	if _, err := exec.LookPath(ociBin); err != nil {
+		glog.Infof("skipping deleteContainersAndVolumes for %s: %v", ociBin, err)
 		return
 	}
 
 	glog.Infof("deleting containers and volumes ...")
 
 	delLabel := fmt.Sprintf("%s=%s", oci.CreatedByLabelKey, "true")
-	errs := oci.DeleteContainersByLabel(oci.Docker, delLabel)
+	errs := oci.DeleteContainersByLabel(ociBin, delLabel)
 	if len(errs) > 0 { // it will error if there is no container to delete
 		glog.Infof("error delete containers by label %q (might be okay): %+v", delLabel, errs)
 	}
 
-	errs = oci.DeleteAllVolumesByLabel(oci.Docker, delLabel)
+	errs = oci.DeleteAllVolumesByLabel(ociBin, delLabel)
 	if len(errs) > 0 { // it will not error if there is nothing to delete
 		glog.Warningf("error delete volumes by label %q (might be okay): %+v", delLabel, errs)
 	}
 
-	errs = oci.PruneAllVolumesByLabel(oci.Docker, delLabel)
+	if ociBin == oci.Podman {
+		// podman prune does not support --filter
+		return
```

Comment:
in this line it seems to return and wont go to the deleteall process?

Reply:
The operation is not supported (volume prune) on podman, so we can only list-and-delete